src/main/java/mate/academy/internetshop/dao/jdbc/OrderDaoImpl.java

```diff
@@ -0,0 +1,164 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.internetshop.dao.OrderDao;
+import mate.academy.internetshop.exception.DataProcessingException;
+import mate.academy.internetshop.lib.anno.Dao;
+import mate.academy.internetshop.model.Order;
+import mate.academy.internetshop.model.Product;
+import mate.academy.internetshop.util.ConnectionUtil;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order create(Order order) {
+        String query = "INSERT INTO internetshop.orders (user_id) "
+                + "VALUES(?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection
+                    .prepareStatement(query, PreparedStatement.RETURN_GENERATED_KEYS);
+            statement.setLong(1, order.getUserId());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                order.setId(resultSet.getLong(1));
+            }
+            addProducts(order);
+            return order;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Unable to execute "
+                    + "create product query. Stack trace: "
+                    + e.getMessage());
+        }
+    }
+
+    @Override
+    public Optional<Order> get(Long id) {
+        String query = "SELECT * "
+                + "FROM internetshop.orders "
+                + "WHERE order_id=?";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(getOrder(resultSet));
+            }
+            return Optional.empty();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Unable to execute "
+                    + "get product query. Stack trace: "
+                    + e.getMessage());
+        }
+    }
+
+    @Override
+    public List<Order> getAll() {
+        List<Order> orders = new ArrayList<>();
+        String query = "SELECT * "
+                + "FROM internetshop.orders;";
```

Comment:
```suggestion
                + "FROM orders;";
```
you setting database name when setting connection

Reply:
Hi! Yep, fixed